Reason: (e) sulu/skeleton 3.0 links the Sulu 2.6 docs on caching with Varnish.
Evidence: https://docs.sulu.io/3.x/cookbook/caching-with-varnish.html answers 200 without a redirect (curl, 2026-10-04).

--- a/config/packages/sulu_http_cache.yaml
+++ b/config/packages/sulu_http_cache.yaml
@@ -6,7 +6,7 @@
 
     # Varnish Configuration:
     #
-    # See: https://docs.sulu.io/en/2.6/cookbook/caching-with-varnish.html
+    # See: https://docs.sulu.io/3.x/cookbook/caching-with-varnish.html
     # Changes in the public/index.php is needed to disable the symfony http cache and use varnish instead
     #
     #        varnish:
